docs/spec/callables.rst

```diff
@@ -3,47 +3,107 @@
 Callables
 =========
 
-Argument defaults
------------------
+Terminology
+-----------
+
+In this section, and throughout this specification, the term  "parameter"
+refers to a named symbol associated with a function that accepts the value of
+an argument (or multiple arguments) passed to the function. The term
+"argument" refers to a value passed to a function when it is called.
+
+In Python, parameters fall into one of three categories: positional-only,
```

Comment:
I would say there are five, aligning with https://docs.python.org/3.10/library/inspect.html#inspect.Parameter.kind. It's mostly a matter of semantics, but I think it's better if we use the same definitions as the runtime inspect module. Below you already have separate sections for `*args` and `**kwargs`, consistent with seeing them as separate kinds.